packages/cxl-jwplayer/src/components/cxl-jwplayer.js

```diff
@@ -0,0 +1,89 @@
+import { LitElement, html, customElement, property } from 'lit-element';
+import '@conversionxl/cxl-lumo-styles';
+import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
+import cxlJwplayerGlobalStyles from '../styles/global/cxl-jwplayer-css.js';
+
+@customElement('cxl-jwplayer')
+export class CxlJwPlayer extends LitElement {
+  @property({ type: Object })
+  player = null;
+
+  @property({ type: Object })
+  config = {};
+
+  @property({ type: String })
+  mediaId = '';
+
+  constructor() {
+    super();
+    this.jwplayer = window.jwplayer || null;
+    // eslint-disable-next-line no-console
+    if (!this.jwplayer) console.error('JWPlayer library not found.');
+  }
+
+  /*
+   * Implemented to avoid default, which attaches a shadowDOM tree byt default.
+   */
+  createRenderRoot() {
+    return this;
+  }
+
+  render() {
+    return html`
+      <h2>cxl-jwplayer</h2>
```

Comment:
what's this header for? Storybook testing?

Reply:
Oh, yes, sorry about that